.doks/values.yaml

```diff
@@ -29,6 +29,10 @@ global:
     protocol: "https"
     disableSSL: "true"
     isPrivateMode: "true"
+    
+  gitlab:
```

Comment:
It is also missing here https://github.com/codacy/chart/blob/master/codacy/values-production.yaml ?